### PR TITLE
MDBOOK: intro to Command Line Tools, Create A New Project

### PR DESCRIPTION
**Pull request by Connoropolous**
These are two more, in a series of short articles...

- [x] What's an app?
- [x] Intro to DNA
- [x] **new:** Intro to Command Line Tools
- [x] **new:** Create a New Project

On their way...
- generating zomes
- writing zome code (lots of sub-articles)
- packaging your app
- embedding your app

### CODE REVIEW

**Comment by thedavidmeister** [comment]
@Connoropolous i just added WIP to the title, drop it when you've finished the tasks :)

helps us visually scan what PRs need attention

**Comment by Connoropolous** [comment]
Oops, my checklist backfired.

Its just meant to show where these fit, but I'm doing them in smaller PRs a
and trying to merge them as I go.

I just wanted to give people the sense that these aren't just scatterplot
articles

File: doc/holochain_101/src/new_project.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Come up with a name for your application, or at least for your project folder.
+
+Copy or type everything after the `$` sign below into your terminal, except replace `your_app_name` with the name you came up with. Press `Enter` to execute the command.
+
+```shell
+$ hcdev init your_app_name
+```
+
+`hcdev` specifies that you wish to use the Holochain command line tools. `init` specifies to use the command for initializing a new project folder. `your_app_name` is an argument you supply as the app, and folder name.
```
Comment (lucksus) [inline doc/holochain_101/src/new_project.md]: Basically all good and fine. I just want to express that this first sentence "hcdev specifies that [..] you use Holochain", and similarly the `$` above, feels like targeted to somebody who does not know how to use a shell.. which most likely would not be an app developer. So as a developer it feels a bit like not meant for me.. ;)

Maybe it's just me and I don't want to criticize your writing style which in general I enjoy very much. So this is not a strong opinion, just a soft expression of what I feel when reading this. Just to provide this as data :)

File: doc/holochain_101/src/new_project.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+You will typically want to start a new Holochain application this way, since creating them all by hand would waste your time.
+
+In your terminal, change directories to one where you wish to initialize a new Holochain app. The command will create a new folder within the current directory for your app.
+
+Come up with a name for your application, or at least for your project folder.
+
+Copy or type everything after the `$` sign below into your terminal, except replace `your_app_name` with the name you came up with. Press `Enter` to execute the command.
+
+```shell
+$ hcdev init your_app_name
```
Comment (lucksus) [inline doc/holochain_101/src/new_project.md]: I would leave that `$` out than you don't need to tell the reader to copy everything else.. ;)

File: doc/holochain_101/src/new_project.md (hunk 1 above)
Comment (philipbeadle) [inline doc/holochain_101/src/new_project.md]: should we give an example of how to name a project? ie use all lowercase, no spaces

Comment (Connoropolous) [reply doc/holochain_101/src/new_project.md]: thanks Philip, yeah, we should likely add a clarification about what pattern to use for a suitable name

**Comment by thedavidmeister** [comment]
@Connoropolous if you write something like "child of #123" in github issues, rather than checklists in the description, it shows up as an epic that automatically tracks your progress as you close them out

**Comment by Connoropolous** [comment]
Cool, thanks

**Pull request merged**